Dockerfile.node10.test

```diff
@@ -36,8 +36,8 @@ RUN apt-get update \
 	&& apt-get install -y aufs-tools rsync iptables wget git \
 	&& rm -rf /var/lib/apt/lists/*
 
-ENV DOCKER_VERSION 1.11.2
-RUN wget -q -O /tmp/docker-${DOCKER_VERSION}.tgz https://get.docker.com/builds/Linux/x86_64/docker-${DOCKER_VERSION}.tgz \
+ENV DOCKER_VERSION 19.03.12
```

Comment:
@Page- there are two blank continuation lines above at lines 15 and 20. There’s a warning emitted that this will become an error, can you please remove these while you’re at it?